### PR TITLE
Include list of supported browsers

### PR DESCRIPTION
**Pull request by Artiume**

### CODE REVIEW

File: general/clients/index.md [hunk 1]
```diff
@@ -12,6 +12,25 @@ Clients connect your devices to your Jellyfin server and let you view your conte
...
+ - last 2 ChromeAndroid versions
+ - last 2 Safari versions
+ - last 2 iOS versions
+ - last 2 Edge versions
+ - Chrome 27
+ - Chrome 38
+ - Chrome 47
+ - Chrome 53
+ - Chrome 56
+ - Chrome 63
```
Comment (MrTimscampi) [inline general/clients/index.md:30]: All these specific Chrome versions are basically for Tizen and WebOS, not really for desktops (though it should still work)

File: general/clients/index.md [hunk 2]
```diff
@@ -12,6 +12,25 @@ Clients connect your devices to your Jellyfin server and let you view your conte
...
 
+## Browsers
+
+Our goal is to provide support for these browsers. 
+
+ - last 2 Firefox versions
+ - last 2 Chrome versions
+ - last 2 ChromeAndroid versions
+ - last 2 Safari versions
+ - last 2 iOS versions
```
Comment (MrTimscampi) [inline general/clients/index.md:23]: Should probably be grouped with the Safari line and changed to "last 2 versions of Safari of MacOS and iOS"

File: general/clients/index.md [hunk 3]
```diff
@@ -12,6 +12,25 @@ Clients connect your devices to your Jellyfin server and let you view your conte
 
 Do you have a client that interfaces with Jellyfin and want to see it listed here? Please [submit a pull request](https://github.com/jellyfin/jellyfin-docs)!
 
+## Browsers
+
+Our goal is to provide support for these browsers. 
+
+ - last 2 Firefox versions
+ - last 2 Chrome versions
+ - last 2 ChromeAndroid versions
```
Comment (MrTimscampi) [inline general/clients/index.md:21]: Should probably be changed to "last 2 Chrome for Android versions"

File: general/clients/index.md (hunk 1 above)
Comment (Artiume) [reply general/clients/index.md:30]: If you know which go with which, i can add them in parenthesis

File: general/clients/index.md (hunk 2 above)
Comment (Artiume) [reply general/clients/index.md:23]: What about older tablets? Would they need back support like chrome does?

Comment (MrTimscampi) [reply general/clients/index.md:23]: If they're still getting iOS updates, they should be good, I think? I don't use Apple products, so I'm not fully sure what their support policy is.

Comment (Artiume) [reply general/clients/index.md]: I refactored the layout a little, tell me what you think

File: general/clients/index.md (hunk 1 above)
Comment (dkanada) [reply general/clients/index.md:30]: We should make a note that we aren't opposed to supporting older browsers through polyfills and such, but it's not a guarantee.

Comment (Artiume) [reply general/clients/index.md:30]: added

File: general/clients/index.md [hunk 4]
```diff
@@ -12,6 +12,24 @@ Clients connect your devices to your Jellyfin server and let you view your conte
...
 Do you have a client that interfaces with Jellyfin and want to see it listed here? Please [submit a pull request](https://github.com/jellyfin/jellyfin-docs)!
 
+## Browsers
+
+Our goal is to provide support for the two most recent versions of these browsers. Older browsers may be supported, but are not guaranteed. 
+ - Firefox
+ - Chrome
+ - Chrome for Android
+ - Safari for MacOS and iOS
+ - Edge
```
Comment (MrTimscampi) [inline general/clients/index.md:22]: For Edge, you can maybe do this:
```suggestion
 - Edge (EdgeHTML and Chromium engines)
```

File: general/clients/index.md (hunk 1 above)
Comment (MrTimscampi) [reply general/clients/index.md:30]: We should probably remove all the Chrome versions and move "Older browsers may be supported, but are not guaranteed." after the list, as something like this:

```markdown
Older browsers may be supported as a result of the needs of specific web-based clients, but full functionality is not guaranteed on their desktop version.
```

**Pull request merged**